modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -137,6 +143,16 @@ template<typename Net> class Params {
         return *this;
     }
 
+    Params<Net>& cfgInputReshape(const std::map<std::string, std::vector<std::size_t>> &r) {
+        desc.reshape_in_shapes = r;
+        return *this;
+    }
+
+    Params<Net>& cfgInputReshape(const std::vector<std::string>& s) {
+        desc.reshape_in_names.assign(s.begin(), s.end());
```

Comment:
y not just `= `?

Reply:
So, it isn't reflected in PR description.

Reply:
Fixed.